.github/workflows/cargo.yml

```diff
@@ -86,4 +77,4 @@ jobs:
     
     # Ensure Cargo.toml's license field is Apache 2.0.
     - name: All Cargo.toml files have a license field
-      run: .tests/cargo-toml-package-license
+      run: cargo make cargo-toml-package-license
```

Comment:
Do we want to reduce this whole file to just `cargo make test` and have that task run all the tests?

1. The downside is that we lose per-test indication in GitHub.
2. One upside is that we don't have to manage which tests are required in GitHub.
3. Another upside is that we could run all tests locally with a single command, making dev easier.